youtube_dl/extractor/common.py

```diff
@@ -43,6 +43,7 @@
     GeoRestrictedError,
     GeoUtils,
     int_or_none,
+    orderedSet,
```

Comment:
I actually read you correctly but screwed up the implementation. I used `sort -i` instead of `sort -f` and didn't look sufficiently closely at the results before committing them, which is the real error. Fixed.